prowler/providers/aws/services/cognito/cognito_user_pool_client_prevent_user_existence_errors/cognito_user_pool_client_prevent_user_existence_errors.py

```diff
@@ -0,0 +1,23 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.cognito.cognito_idp_client import cognito_idp_client
+
+
+class cognito_user_pool_client_prevent_user_existence_errors(Check):
+    def execute(self):
+        findings = []
+        for pool in cognito_idp_client.user_pools.values():
+            for user_pool_client in pool.user_pool_clients.values():
+                report = Check_Report_AWS(self.metadata())
+                report.region = pool.region
+                report.resource_id = user_pool_client.id
+                report.resource_arn = pool.arn
```

Comment:
Would not be the user_pool_client.arn and user_pool_client.region?

Reply:
Done